### DISCUSSION TITLE
getting error try setting KUBERNETES_MASTER environment variable coredns Containercreating state

### DISCUSSION THREAD

**Discussion by amarjothi** [coredns/coredns#4554] [General]
Warning  FailedCreatePodSandBox  70s (x131 over 29m)  kubelet            (combined from similar events): Failed to create pod sandbox: rpc error: code = Unknown desc = failed to setup network for sandbox "f049cd07dd737cc430ea2186c035d168ac7087bf1700599c79ed2c9dcfa94545": invalid configuration: no configuration has been provided, try setting KUBERNETES_MASTER environment variable
[r

**Comment by chrisohaver** [comment]
https://github.com/kubernetes/kubernetes/issues/100719

**Comment by chrisohaver** [comment]
Answered in cross-posted issue: https://github.com/kubernetes/kubernetes/issues/100719#issuecomment-824940815
